src/__snapshots__/index.test.ts.snap

```diff
@@ -2136,7 +2267,7 @@ async function putEcho(
   const response = await axios(Object.assign({
     baseURL: 'http://192.168.10.149:8000/v3',
     paramsSerializer: querystring.stringify.bind(querystring),
-    validateStatus: status => 200 <= status && 300 > status,
+    validateStatus: (status: number) => 200 <= status && 300 > status,
```

Comment:
Looking at this, I think that the `filterStatuses` option could be resused to make it sync with the types one. One could just like to have any status spawned here (and in fact, it would be a saner way to handler errors). I thin it can be done in another PR, merging as is for now.